docker-compose.override.yml

```diff
@@ -0,0 +1,23 @@
+version: "3.4"
+
+services:
+
+    python2:
+        build:
+            target: cfgov-dev
+        ports:
+            - "8000:8000" # Django
+        volumes:
+            - ./:/src/cfgov-refresh
+            - ./develop-apps:/src/develop-apps
```

Comment:
Something feels wrong about this line, although I realize you haven't changed it in this PR. The [`extend-environment.sh`](https://github.com/cfpb/cfgov-refresh/blob/688a818670a86c4f34d230f6bfd82f4eaccfb110/extend-environment.sh#L3) script adds apps that are in /src/cfgov-refresh/develop-apps/ to the path, not apps that are in /src/develop-apps. And since the entire source tree gets added under /src/cfgov-refresh anyway, isn't it unnecessary to add another volume specifically for develop-apps? Perhaps this line (and the similar one below) can be removed?

Reply:
Agreed.  That _seems_ broken.  I'll check it out.

Reply:
Discussed with @chosak that we don't even need a separate volume here.  It's been removed, and confirmed that `PYTHONPATH` is still built correctly for `develop-apps`.